examples/applications/Face&EyesDetection.py

```diff
@@ -0,0 +1,42 @@
+'''
+steps to exicute this application:
+1. exicute the WifiCam.ion file in ESP32 cam module which is available at examples->WifiCam->WifiCam.ion.
+2. before exicution make shure that the wifi cretentials are saves accordingly.
+3. After uploading the code to the esp32 restart the esp32 cam module.
+4. open the serial monitor and note down the IP address and chane the 18th line of this code with that ip address
+5. Now exicute this python code in local computer and make shure that the computer is also connected to the same netwok
+6. Now you can view the camera output in the windows poped by the python and see the face and ee detection working in real time.
+
+'''
+
+import cv2
+import urllib.request
+import numpy as np
+ 
+f_cas= cv2.CascadeClassifier(cv2.data.haarcascades+'haarcascade_frontalface_default.xml')
+eye_cascade=cv2.CascadeClassifier(cv2.data.haarcascades +'haarcascade_eye.xml')
+url='http://xxx.xxx.xxx.xxx/cam-lo.jpg' #change the IP address to the IP address to which the ESP32 cam is connected to
```

Comment:
This should be abstracted as a command line flag. Don't ask user to modify code.